country_config/null_download.json

```diff
@@ -5,34 +5,36 @@
 
   "rawIntro":"Use this form to download raw data for use in your own analysis. Select which form's data you would like to download, and optionally further select which attributes you would like to download. ",
 
-	"prepared":[{
-		"title":"Consultations",
-		"api":"/api/export/data_table/test/reg_1",
-		"access":"cd",
-		"variables":[
-			["reg_1","consultations"]
-		],
-		"group_by": [
-			["epi_year", "Year"],
-			["clinic:location", "Clinic"],
-			["epi_week", "Week"]
-		]
-		
-
-	},{
-      "title":"Communicable Diseases",
-      "api":"/api/export/category/demo_case/cd_tab/communicable_diseases",
-      "name_category": "individual_icd",
-      "access":"cd",
-      "fields":[
-          ["intro./visit","Visit Type"],
-          ["pt./visit_date$date","Visit Date"],
-          ["pt./visit_date$year","Year"],
-          ["pt./visit_date$month","Month"],
-          ["pt./visit_date$epi_week","Epi Week"],
-          ["region","Region"],
-          ["district","District"],
-          ["clinic","Clinic"],
+    "prepared":[
+        {
+	    "title":"Completeness",
+	    "api":"/api/export/week_level/test/clinic",
+	    "access":"cd",
+            "wide_data_format":1,
+	    "variable":
+                ["completeness:/completeness/reg_1/1/4/<start_week>/5,6/reg_1/<end_date>", "Completeness"]
+	    
+        },  {
+	    "title":"Cases",
+	    "api":"/api/export/week_level/test/clinic",
+	    "access":"cd",
+            "wide_data_format": 1,
+	    "variable": ["tot_1", "tot_1", "Cases"]
```

Comment:
Every other variable field only has two elements, and it seems very odd that "tot_1" should be repeated. I don't really understand how this works, but just want to do a sanity check that this is not a typo?

Reply:
It is not a typo, but not very clear. When asking for variable in this way it is also possible to restrict by a separate variable. I.e count female cases among patients with hypertension.

Reply:
Why not use here another JSON object instead of array. E.g.
```
"variable": {
   "foo": "tot_1",
   "bar": "tot_1",
   "baz": "Cases"
}
```
Maybe it's not as hard change as it seems to me at the moment.

Reply:
We could, essentially just following the convention in the rest of this config file. Will be quite a lot of work sorting this out. In principle I agree that this is a decent proposal, maybe make a separate issue about this?